test/test.js

```diff
@@ -85,19 +85,21 @@ describe('Routes', () => {
             'Content-Type': 'application/json'
           },
           body: JSON.stringify({
-            success: true
+            success: true,
+            walletAddress: 'abc'
```

Comment:
Can we use test data that at least resembles FIL addresses? When I encounter the value `abc`, I'll have no idea what is it connected to.

For example, we can use `f1abc` or `f1test` instead. 

I think we will need to switch to real FIL addresses in the future and reject HTTP requests with invalid wallet address. To prepare for that, it would be great to extract this dummy test wallet address into a shared constant used by all tests.

Reply:
good idea!